apps/content/lib/cms/http_client.ex

```diff
@@ -73,13 +75,21 @@ defmodule Content.CMS.HTTPClient do
   end
 
   # Convert nested key values to their own keys, if whitelisted
-  @spec list_to_params(String.t(), param_list, {safe_key(), String.t()}) :: param_list
+  @spec list_to_params(String.t(), param_list, {safe_key(), String.t()} | map) :: param_list
   defp list_to_params(key, acc, {sub_key, sub_val}) when sub_key in @safe_keys do
     stringify_params({key <> "[#{sub_key}]", sub_val}, acc)
   end
 
+  # Nested params from 302 redirects come in as a single-pair Map.t() element
+  defp list_to_params(key, acc, val) when is_map(val) do
```

Comment:
when the test runs, this function is never called

Reply:
See latest commit. Turns out we no longer need this clause :)